src/Microsoft.FeatureManagement/Targeting/ContextualTargetingFilter.cs

```diff
@@ -5,7 +5,6 @@
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
 using System;
-using System.Collections.Generic;
```

Comment:
Unrelated change?

Reply:
Yes, an unused dependency. I can leave it out of this PR if you prefer

Reply:
yes please